pages/common/neo4j-admin.md

```diff
@@ -0,0 +1,32 @@
+# neo4j-admin
+
+> Manage and administer a Neo4j DBMS.
+> More information: <https://neo4j.com/docs/operations-manual/current/tools/neo4j-admin/>.
+
+- Start the Neo4j DBMS:
+
+`neo4j-admin server start`
+
+- Stop the Neo4j DBMS:
+
+`neo4j-admin server stop`
+
+- Set the initial password of the default `neo4j` user (required before the first DBMS start):
+
+`neo4j-admin dbms set-initial-password {{database_name}}`
+
+- Create an archive (dump) of an offline database to a file named `database_name.dump`:
+
+`neo4j-admin database dump --to-path={{path/to/dumps}} {{database_name}}`
```

Comment:
What does `path/to/dumps` mean?
If you mean a folder which you probably mean according to [this](https://neo4j.com/docs/operations-manual/current/kubernetes/operations/dump-load/#kubernetes-neo4j-load) and [this](https://neo4j.com/docs/operations-manual/current/backup-restore/restore-dump/) wouldn't be better to use `path/to/folder`?
Also in all the examples in the docs I have seen the path isn't relative.
```suggestion
`neo4j-admin database dump --to-path={{/path/to/folder}} {{database_name}}`
```